README.md

```diff
@@ -38,6 +38,13 @@ WIP team site
 - https://github.com/wmde/Number
 - https://github.com/wmde/Time
 - https://github.com/wmde/git-filter-repo-docker-action
+- https://github.com/wmde/vuex-helpers
+- https://github.com/wmde/lib-version-check
+- https://github.com/wmde/hamcrest-html-matchers
+- https://github.com/wmde/mediawiki-doctrine-connection
+- https://github.com/wmde/WikidataApiGem
+- https://github.com/wmde/wbs_propertypairs
+- https://github.com/wmde/prototype-wikibase-graphql-api
```

Comment:
not sure this fits the intention of the list. It's very much a prototype and if it has served its purpose it should be archived even.

Reply:
I had the same thought over at https://github.com/wmde/wikidata-wikibase-team/pull/3#issuecomment-800511977 regarding some of the repositories there.

As this is "just a list" for now I think we could include them.
In a future iteration I think this list should include more categorizations of the repos / taggings.

Reply:
I was about to remove it from the list, but then @addshore suggested to include "all the things". Keeping in the list, maybe the repo gets archived meanwhile and then will drop off the list.